lib/common_test/src/ct_telnet.erl

```diff
@@ -587,6 +680,26 @@ returns `HaltReason = {nnn,["NNN"]}`.
 Options `repeat` and `sequence` can be combined to match a sequence multiple
 times.
 """.
+-spec expect(Connection, Patterns, Opts) -> {'ok', Match} | {'ok', Match, HaltReason}
+              | {'error', Reason} | no_return()
+              when Connection :: connection(),
+                   Patterns :: Pattern | [Pattern],
+                   Pattern :: PatternString | {Tag, PatternString},
+                   Tag :: term(),
+                   PatternString :: unicode:charlist(),
+                   Opts :: [Option],
+                   Option :: {'idle_timeout', non_neg_integer()}
+                           | {'total_timeout', non_neg_integer()} | 'ignore_prompt'
+                           | 'no_prompt_check' | 'wait_for_prompt' | 'repeat'
+                           | {'repeat', non_neg_integer()} | 'sequence' | {'halt', Patterns},
+                   Match :: MatchResult | [MatchResult],
+                   MatchResult :: [CaptureData | {Tag, CaptureData}]
+                                | [[CaptureData | {Tag, CaptureData}]],
+                   CaptureData :: string()
```

Comment:
I think this should be fixed in code, so both {'error', string(), binary()} and {'incomplete', string(), binary()} are not possible.

Reply:
Fixed in documentation.